packages/split-button/src/SplitButton.ts

```diff
@@ -72,16 +72,14 @@ export class SplitButton extends SizedMixin(PickerBase) {
     protected listRole: 'listbox' | 'menu' = 'menu';
     protected itemRole = 'menuitem';
 
-    public focus(): void {
-        if (this.disabled) {
-            return;
+    public get focusElement(): HTMLElement {
+        if (this.open) {
+            return this.optionsMenu;
         }
         if (this.left) {
-            this.trigger.focus();
-            return;
+            this.trigger;
```

Comment:
Is there meant to be a return here?

```suggestion
            return this.trigger;
```

Reply:
Bwah! I was just like "yeah, these tests are supposed to change..." 😞